port: /dev/PICOBEE
baud: 921600
samples: 2
min_range: 0.02
max_range: 0.6
flip_terabees: False # if this is true, left is right and right is left (in case they are wired backwards)
# 2.011
